### PR TITLE
Upgrading to ruby 2.7.2

### PR DESCRIPTION
**Pull request by boris-ning-usds**
- Upgrading to ruby 2.7.2

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -28,7 +28,7 @@ DEPENDENCIES
   thin
 
 RUBY VERSION
-   ruby 2.5.1p57
+   ruby 2.7.2p137
 
 BUNDLED WITH
    1.17.3
```
Comment (TomNUSDS) [inline Gemfile.lock]: Maybe `BUNDLED WITH` should be 2.1.4?

**Review by TomNUSDS — COMMENTED** [review]
I don't know much about ruby, but searching github for Gemlock files with a`RUBY VERSION 2.7.2p137` shows most "BUNDLED WITH" to be 2.1.4.

**Review by lisac-usds — APPROVED** [review]
can confirm this configuration runs on my local setup!

File: Gemfile.lock (hunk 1 above)
Comment (lisac-usds) [reply Gemfile.lock]: locally, I was able to upgrade from Bundler1 to Bundler2 (in my case: `2.2.16`). I don't know enough about ruby and bundler as to make a rec on whether this should be part of this PR. 
(ref: [How to Upgrade to Bundler 2](https://bundler.io/guides/bundler_2_upgrade.html)).

Comment (boris-ning-usds) [reply Gemfile.lock]: Yep, updated to bundler 2 via `bundle update --bundler`
```
Using bundler 2.1.4
Using daemons 1.3.1
Using eventmachine 1.2.7
Using ruby2_keywords 0.0.4
Using mustermann 1.1.1
Using rack 2.2.3
Using rack-protection 2.1.0
Using rack-ssl-enforcer 0.2.9
Using tilt 2.0.10
Using sinatra 2.1.0
Using thin 1.8.0
Warning: the lockfile is being updated to Bundler 2, after which you will be unable to return to Bundler 1.
Bundle updated!
```

Reran the code and onceler is still working just fine.

**Comment by lisac-usds** [comment]
update: we've deployed this change, and things look good. 👍

**Pull request merged**